guide/creating-your-bot/common-questions.md

```diff
@@ -39,6 +39,14 @@ if (member.roles.exists('name', '<role name>')) {
 }
 ```
 
+### How do I limit a command to a single user?
+
+```js
+if (message.author.id === '<id>') {
```

Comment:
I somewhat recommend using the inverse as there's less chance of bracket worries for beginners.

Reply:
I'm on the fence about this simple because of how the question is asked and how the code should reflect the question, and vice versa, and also because of how simple it looks. Consistency is a good point, but I think that would apply more to the pages with full explanations, rather than these snippets.